tests/integration-service/integration.go

```diff
@@ -1,47 +1,41 @@
 package integration
 
 import (
-	//	"bytes"
-	//	"context"
 	"fmt"
-	//	"net/http"
 	"strings"
 	"time"
 
 	"github.com/devfile/library/pkg/util"
 	"github.com/google/uuid"
-	appservice "github.com/redhat-appstudio/application-service/api/v1alpha1"
 	"github.com/redhat-appstudio/e2e-tests/pkg/constants"
 	"github.com/redhat-appstudio/e2e-tests/pkg/utils"
-	"github.com/tektoncd/pipeline/pkg/apis/pipeline/v1beta1"
+	"github.com/redhat-appstudio/e2e-tests/pkg/framework"
+	"k8s.io/apimachinery/pkg/api/errors"
 
-	//	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
-	//appstudioshared "github.com/redhat-appstudio/managed-gitops/appstudio-shared/apis/appstudio.redhat.com/v1alpha1"
+	appstudioshared "github.com/redhat-appstudio/managed-gitops/appstudio-shared/apis/appstudio.redhat.com/v1alpha1"
+	v1 "k8s.io/api/core/v1"
+	klog "k8s.io/klog/v2"
 
 	. "github.com/onsi/ginkgo/v2"
 	. "github.com/onsi/gomega"
-	"github.com/redhat-appstudio/e2e-tests/pkg/framework"
-	v1 "k8s.io/api/core/v1"
-	"k8s.io/apimachinery/pkg/api/errors"
-	klog "k8s.io/klog/v2"
-	//	routev1 "github.com/openshift/api/route/v1"
 )
 
 const (
 	containerImageSource = "quay.io/redhat-appstudio-qe/busybox-loop:latest"
 	gitSourceRepoName    = "devfile-sample-python-basic"
 	gitSourceURL         = "https://github.com/redhat-appstudio-qe/" + gitSourceRepoName
+	bundleURL            = "quay.io/kpavic/test-bundle:build-pipeline-pass"
```

Comment:
We created the official repository for the example bundle images at [`quay.io/redhat-appstudio/example-tekton-bundle`](https://quay.io/repository/redhat-appstudio/example-tekton-bundle), so it would probably be best if we used `quay.io/redhat-appstudio/example-tekton-bundle:build-pipeline-pass` here.

Reply:
Updated, thanks!